fre/make/gfdlfremake/buildBaremetal.py

```diff
@@ -120,10 +132,18 @@ def run(self):
         Param:
             - self : The dockerfile object
         """
-###### TODO make the Makefile
-        command = [self.bld+"/compile.sh","|","tee",self.bld+"/log.compile"]
-        try:
-            subprocess.run(args=command, check=True)
-        except:
-            print("There was an error running "+self.bld+"/compile.sh")
-            raise
+        command = [self.bld+"/compile.sh"]
+
+        # Run compile script
+        p1 = subprocess.Popen(command, stdout=subprocess.PIPE,stderr=subprocess.STDOUT)
+
+        # Direct output to log file as well
+        p2 = subprocess.Popen(["tee",self.bld+"/log.compile"], stdin=p1.stdout)
+
+        # Allow p1 to receive SIGPIPE is p2 exits
+        p1.stdout.close()
+        p2.communicate()
+
+        if p1.returncode != 0:
+            print("\nThere was an error running {self.bld}/compile.sh")
```

Comment:
I think this needs to be something like
```
print("\nThere was an error running " + self.bld + " /compile.sh")
```

Reply:
Oooop, thanks! Forgot the f in the f-print statement. I'll fix it